docs/correlating-traces-and-logs.md

```diff
@@ -0,0 +1,132 @@
+# Correlating Trace and Logs
+
+The [OpenTelemetry trace API] can be used to extract trace metadata from a
+context containing a span. That metadata can then be used to annotate log
+events so they will be correlated with the trace.
+
+## Extracting Trace Metadata
+
+To extract trace metadata, use the [`SpanContextFromContext`] function. This
+will extract the trace metadata from an [`context.Context`] and return it in
+the form of a [`SpanContext`].
+
+```go
+spanContext := trace.SpanContextFromContext(ctx)
+if !spanContext.IsValid() {
+	// ctx does not contain a valid span.
+    // There is no trace metadata to add.
+    return
+}
+```
+
+## Annotating Log Events
+
+Once you have a [`SpanContext`], you can use the trace metadata it contains to
+annotate log events. How this is done will depend on the logging library used.
+
+### Structured Logging
```

Comment:
@MrAlias "Structured" as in relying on data structure, or a structured text output, such as TOML/YAML/JSON? I believe Log Observer has an easier time digesting JSON logs.

Reply:
Here "structured" refers to the data structure and the annotated attributes added to a log event. By contrast, an unstructured logger will usually only accept a log message as a string.